cephfs/file.go

```diff
@@ -255,3 +255,38 @@ func (f *File) Fallocate(mode FallocFlags, offset, length int64) error {
 	ret := C.ceph_fallocate(f.mount.mount, f.fd, C.int(mode), C.int64_t(offset), C.int64_t(length))
 	return getError(ret)
 }
+
```

Comment:
You can create a `LockOp` type (int) and make `LockSH` and the other constants a `LockOp`.